docs/src/hohqmesh_tutorial.md

```diff
@@ -0,0 +1,481 @@
+# Unstructured meshes with HOHQMesh.jl
+
+Trixi.jl supports numerical approximations on unstructured quadrilateral meshes
+with the `UnstructuredMesh2D` mesh type.
+
+The purpose of this tutorial is to demonstrate how one can use the `UnstructuredMesh2D`
+functionality of Trixi.jl. This begins by running and visualizing an available unstructured
+quadrilateral mesh example. Then, the tutorial will demonstrate how one can
+conceptualize a problem with curved boundaries, generate
+a curvilinear mesh using the available software in the Trixi.jl ecosystem,
+and then run a simulation using Trixi.jl on said mesh.
+
+Unstructured quadrilateral meshes can be made
+with the [*High-Order Hex-Quad Mesh (HOHQMesh)*](https://github.com/trixi-framework/HOHQMesh)
+generator created and developed by David Kopriva.
+HOHQMesh is a mesh generator specifically designed for spectral element methods where elements can be larger (due to the high accuracy
+of the spatial approximation) and provides high-order boundary curve information (needed to accurately set boundary conditions).
+For more information about the design and features of HOHQMesh one can refer to its
+[official documentation](https://trixi-framework.github.io/HOHQMesh/).
+
+HOHQMesh is incorporated into the Trixi.jl framework via the registered Julia package
+[HOHQMesh.jl](https://github.com/trixi-framework/HOHQMesh.jl).
+This package provides a Julia wrapper for the HOHQMesh generator that allows users to easily create mesh files without the need to build
+HOHQMesh from source. To install the HOHQMesh package execute
+```julia
+julia> import Pkg; Pkg.add("HOHQMesh")
+```
+Now we are ready to generate an unstructured quadrilateral mesh that can be used by Trixi.jl.
+
+## Running and visualizing an unstructured simulation
+
+Trixi.jl supports solving hyperbolic problems on several mesh types.
+There is a default example for this mesh type that can be executed by
+```julia
+julia> trixi_include(default_example_unstructured())
+```
+This will compute a smooth, manufactured solution test case for the 2D compressible Euler equations
+on the curved quadrilateral mesh described in the
+[Trixi documentation](https://trixi-framework.github.io/Trixi.jl/stable/meshes/unstructured_quad_mesh/).
+
+Apart from the usual error and timing output provided by the Trixi.jl run, it is useful to visualize and inspect
+the solution. Currently, for solutions on unstructured quadrilateral meshes, this requires post-processing the
+Trixi.jl output file(s) using the [`Trixi2Vtk`](https://github.com/trixi-framework/Trixi2Vtk.jl) tool
+and plotting them with [ParaView](https://www.paraview.org/download/).
```

Comment:
Since we will have support for other options soon, what about re-writing this such that Paraview is described as a possibility, not a necessity?

Reply:
That is true, e.g., now that the `Makie.jl` PRs are converging. I wasn't sure the best way to present this. Is it the case that Makie should supplant `Trixi2Vtk`? I.e, is the plan to deprecate `Trixi2Vtk` in the future?

Reply:
No, IMHO Trixi2Vtk will never be deprecated, simply because of the ability to do visualization *and* postprocessing in ParaView directly.

Reply:
Trixi2Vtk and visualization via Plots/Makie are two (three) tools with similar, but slightly different purposes.

Reply:
okay that is good to know. This is a product of my own confusion because I only use the ParaView approach

Reply:
Trixi2Img is deprecated

Reply:
Okay, I updated the wording here to clarify that this is one option, not the only option, for unstructured mesh solutions